src/tests/multihost/basic/test_kcm.py

```diff
@@ -168,3 +168,19 @@ def test_ssh_forward_creds(self, multihost, enable_kcm):
             if 'KCM:14583109' in line:
                 has_cache = True
         assert has_cache is True
+
+    def test_kvno_display(self, multihost, enable_kcm):
+        """
+        Test kvno correctly displays vesion numbers of principals
+        #https://pagure.io/SSSD/sssd/issue/3757
+        """
+        ssh = SSHClient(multihost.master[0].sys_hostname,
+                        username='foo4', password='Secret123')
+        host_princ = 'host/%s@%s' % (multihost.master[0].sys_hostname,
+                                     'EXAMPLE.TEST')
+        kvno_cmd = 'kvno %s' % (host_princ)
+        (stdout, _, exit_status) = ssh.execute_cmd(kvno_cmd)
+        for line in stdout.readlines():
+            assert '%s: kvno = 2' % (host_princ) in line
```

Comment:
'kvno = 2' will be returned for a system where the host key way not updated after the join. If you can be sure about this in the test than testing for '= 2' is ok. But I think it would be sufficient to check if there is a proper positive unsigned 32-bit integer.